packages/scatterplot/index.d.ts

```diff
@@ -16,6 +16,33 @@ declare module '@nivo/scatterplot' {
         color: string
     }
 
+    interface Point {
+        id: string
+        data: {
+            x: number
+            y: number
+        }
+    }
+
+    export type SymbolSvg = (arg: {
+        x: number
+        y: number
+        size: number
+        color: string
+        onMouseEnter?: ScatterPlotMouseHandler
+        onMouseMove?: ScatterPlotMouseHandler
+        onMouseLeave?: ScatterPlotMouseHandler
+        onClick?: ScatterPlotMouseHandler
+        point: Point
+    }) => React.ReactNode
+
+    export type SymbolCanvas = (
```

Comment:
For other components, when allowing to customize canvas rendering, I used this approach:

```typescript
export type SymbolCanvas = (ctx: CanvasRenderingContext2D, {
    // ...props
}) => void
```